indev/libinput.c

```diff
@@ -250,6 +267,83 @@ void libinput_read(lv_indev_drv_t * indev_drv, lv_indev_data_t * data)
   libinput_read_state(&default_state, indev_drv, data);
 }
 
+libinput_lv_event_t *new_event(libinput_drv_state_t *state)
+{
+  libinput_lv_event_t *evt = &state->points[state->end];
+
+  if (++state->end == MAX_EVENTS)
```

Comment:
Would you also have to do something to `state->start` here to prevent `end` overtaking `start` which would then read later events first when accessing `start`?